apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -1,3 +1,4 @@
+import clsx from "clsx";
```

Comment:
create a new helper `getValidaClasses` for it

Reply:
I'm not sure I understand correctly why do we need it. For readability improve? (for me it's so much easier to read the first version with filter and join) 

To refactor this?
```
const inputClassNames = clsx(
	styles["input-field"],
	leftIcon && styles["with-left-icon"],
	rightIcon && styles["with-right-icon"],
);
```

Should it be something like this?
```
import clsx from "clsx";

type Properties = {
	hasError: boolean;
};

const getValidClassNames = ({ hasError }: Properties): string => {
	return clsx({
		"is-invalid": hasError,
		"is-valid": !hasError,
	});
};

export { getValidClassNames };
```

Reply:
no, this should be just a wrapper for clsx, like
```javascript
const getValidClassNames = (...inputs) => {
	return clsx(...inputs);
};
```

Reply:
I meant something like this 🙂 Instead of importing `clsx ` all the time 

```ts
import { type ClassValue, clsx } from "clsx";

const getValidClassNames = (...inputs: ClassValue[]): string => {
	return clsx(...inputs);
};

export { getValidClassNames };
```